src/routes/auth.route.js

```diff
@@ -7,5 +7,6 @@ router.post(
 	'/signup',
 	AuthController.signup,
 );
+router.get('/activate', AuthController.verifyUser);
```

Comment:
Here you are updating but you have used the `get` method. Instead, you can use `patch`

Reply:
Done